app/models/concerns/classifiable.rb

```diff
@@ -0,0 +1,34 @@
+module Classifiable
```

Comment:
Is this tested anywhere like in another class as side affect?

Reply:
And as it gets rendered in the views